styles/styleViews.css

```diff
@@ -1,6 +1,6 @@
 body {
```

Comment:
This is a second body tag. You should only have one and it should be inside of your base.css file.